### ISSUE TITLE
Default `iteration` about SPIN. (Reward model~Policy model)

### ISSUE THREAD

**Issue by KyujinHan** [lucidrains/self-rewarding-lm-pytorch#9]
**What is the iteration value calculated for SPIN loss? (Paper mentioned)**
Where does that appear in the code, and is it something we can control?

Could you let me know?

Thanks.

**Comment by lucidrains** [comment]
@KyujinHan yea, you can do multiple SPIN interations by [this hyperparameter](https://github.com/lucidrains/self-rewarding-lm-pytorch/blob/main/self_rewarding_lm_pytorch/self_rewarding_lm_pytorch.py#L569)

eventually i want to make it so one can order the fine tuning arbitarily (say SPIN - SPIN - SFT - Self-Reward) . this was an idea suggested by @teknium1 in a private discord.
